packages/yourloops/components/patient-list/models/enums/patient-list.enum.ts

```diff
@@ -37,7 +37,7 @@ export enum PatientListColumns {
   Flag = 'flag',
   Gender = 'gender',
   GlucoseManagementIndicator = 'glucose-management-indicator',
-  Hypoglycemia = 'hypoglycemia',
+  BelowRange = 'below-range',
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**🤓 nitpick:** ‏Values are ordered alphabetically here 👼